app/views/users/index.html.erb

```diff
@@ -1,3 +1,3 @@
-<h1>List of Users</h1>
-<p><%= @placeholder_text %></p>
-<p>Find me in app/views/users/index.html.erb</p>
+<section id="users">
+  <%= render partial: 'user', collection: @users %>
+</section>
```

Comment:
- [ ] Great job working on the views so far but I think it would be great if you make like the given wireframe since you should at least make the same layout so kindly adjust your element the image in a box and the name of user with no of posts in the right so please make them the same given wireframe 🔥💪🏻🎉


![image](https://github.com/richie1988/blog-app/assets/58553711/ead20f39-8e2f-4e77-b06f-e530d1fe39b7)


- Your
![image](https://github.com/richie1988/blog-app/assets/58553711/359f8401-1a97-4706-bf0b-07d43c2604ec)


- Given
![image](https://github.com/richie1988/blog-app/assets/58553711/cc6d19e9-b104-40fa-81e9-7676a30ab178)